Frontend/src/components/Register.js

```diff
@@ -0,0 +1,245 @@
+import React, { useState } from "react";
+import { useForm } from 'react-hook-form';
+import {useHistory} from "react-router-dom";
+import { yupResolver } from '@hookform/resolvers/yup';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JeremiasFuentes/unq-desapp-fuentes-diaz/pullRequest?prid=8198099)
```suggestion
import { yupResolver } from "@hookform/resolvers/yup";
```